lib/foreman_tasks/tasks/export_tasks.rake

```diff
@@ -221,6 +222,15 @@ namespace :foreman_tasks do
         tasks.order('started_at desc').all.each do |task|
           html << "<tr><td><a href=\"#{task.id}.html\">#{task.label}</a></td><td>#{task.started_at}</td>\
                    <td>#{task.state}</td><td>#{task.result}</td></tr>"
+        rescue StandardError => e
+          if skip_errors
+            puts _('Skipping indexing a task that failed to export because SKIP_FAILED is set to "true"')
+          else
+            puts _('Indexing tasks failed because the task below failed to be indexed.')
+            puts _('Re-run with SKIP_FAILED=true if you wish to prevent this problem from halting task export altogether.')
+            puts task.inspect
```

Comment:
Do we expect that there could be a task which would pass the regular export but would fail when generating the index (or the other way around)? If not, then we would be logging basically the same failure twice

Reply:
Now that generate_index() will only receive successfully exported tasks, I think the `rescue` clause here is OK, too.